src/atoms/Button/buttons.module.scss

```diff
@@ -63,9 +63,9 @@ $brand-white: color('primary-2');
 }
 
 .toggle-selected {
-  color: $brand-white;
-  border: 2px solid $selected-color;
-  background-color: $selected-color;
+  color: color('primary-1');
```

Comment:
Under [the new design](https://www.figma.com/file/iVP8PicuOCAkB7NpswyAWK/Mortar---RCL-color-updates?node-id=38%3A43&viewport=-299%2C-119%2C0.6018280982971191), selected toggle buttons should now look slightly fancier. This helps distinguish between the main CTA and ordinary button components.
![Feb-04-2021 21-17-01](https://user-images.githubusercontent.com/6582067/106980665-96238980-672e-11eb-9f02-f83efa027601.gif)

Reply:
Appreciate the screenshots here.